src/main/java/org/opensearch/geospatial/ip2geo/common/Ip2GeoLockService.java

```diff
@@ -43,6 +43,19 @@ public Ip2GeoLockService(final ClusterService clusterService, final Client clien
         this.lockService = new LockService(client, clusterService);
     }
 
+    /**
+     * Constructor
+     *
+     * @param clusterService the cluster service
+     */
+    public Ip2GeoLockService(final ClusterService clusterService) {
+        this.clusterService = clusterService;
```

Comment:
We might need to add lockService not initialized check in every Ip2GeoLockService methods.

Reply:
It should never happen (i.e. onNodeStarted will always be called and the lock service will get set accordingly) but these checks would be good to put in none-the-less.

Reply:
Added the checks